flask_openapi3/openapi.py

```diff
@@ -216,18 +223,40 @@ def api_doc(self) -> Dict:
         # Set paths
         spec.paths = self.paths
 
+        # Add ValidationErrorResponseModel to components schemas
+        self.components_schemas[ValidationErrorResponseModel.__name__] = Schema(**ValidationErrorResponseModel.schema())
```

Comment:
Shouldn't this be addable by the user when initializing the application? What if the user doesn't use validation but chooses to use your library?

I agree that it's a strange/weird scenario if he chooses that though.